src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx

```diff
@@ -94,6 +103,7 @@ export const getTopQueriesColumns = (): Column<KeyValueRow>[] => {
         readRowsColumn,
         readBytesColumn,
         userSIDColumn,
+        durationColumn,
```

Comment:
It's better to place `durationColumn` near `endTimeColumn` (before or after)

Reply:
done